core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -188,6 +196,7 @@ public void detach(PlayerView playerView) {
         surfaceHolderRequester = null;
         listenersHolder.removeStateChangedListener(playerView.getStateChangedListener());
         listenersHolder.removeVideoSizeChangedListener(playerView.getVideoSizeChangedListener());
+        this.playerView = null;
```

Comment:
We should unset the `SubtitleRendererOutput` here as it references the playerView.